airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -18,7 +18,8 @@ dependencies {
     implementation project(':airbyte-queue')
 
     integrationTestImplementation project(':airbyte-integrations:bases:standard-destination-test')
-    integrationTestImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
+    // depends on normalization docker image creation, which is handled in composeBuild.
+    integrationTestImplementation files(task('composeBuild').outputs)
```

Comment:
unfortunately i think this is pretty intolerable. i think it pushes me far enough to think that we should scrap this approach.

Reply:
this is needed because now the only way that the normalization image gets built is by `composeBuild` instead of `:airbyte-integrations:bases:base-normalization:airbyteDocker`. We _could_  keep the `airbyteDocker` task as well for normalization, but then we'd have 2 sources of truth for how the container gets built which also seems really terrible.